scRNA-seq_module_cheatsheet (#140)
AlexsLemonade/training-modules: module_cheatsheets/scRNA-seq-cheatsheet.md

| `scran`                | [`quickCluster()`](https://www.rdocumentation.org/packages/scran/versions/1.0.3/topics/Quick%20clustering)                | Quick Clustering     | Groups similar cells into clusters which are stored in the `SingleCellExperiment` object and are used for the calculation of size factors by `scran::computeSumFactors`                                |
| `scran`                | [`computeSumFactors()`](https://www.rdocumentation.org/packages/scran/versions/1.0.3/topics/Deconvolution%20Methods)                 | Compute Sum Factors        | Returns a numeric vector of computed sum factors for each cell cluster stored in the `SingleCellExperiment` object. The cluster-based size factors are deconvolved into cell-based size factors that are stored in the `SingleCellExperiment` object and used by the `scran::normalize` function for the normalization of each cell's gene expression profile.                   |

> | `scater`               | [`normalize()`](https://www.rdocumentation.org/packages/scater/versions/1.0.4/topics/normalize)                       | Normalize      | Returns the SingleCellExperiment object with normalized expression values for each cell, using the appropriate size factor and log transformation  |

Reviewer: ```suggestion
| `scater`               | [`normalize()`](https://www.rdocumentation.org/packages/scater/versions/1.0.4/topics/normalize)                       | Normalize      | Returns the `SingleCellExperiment` object with normalized expression values for each cell, using the size factors stored in the object  |
```

Author: Got it, thanks!
